test/github-issues/1465/save-relation.ts

```diff
@@ -0,0 +1,39 @@
+import "reflect-metadata";
+
+import { assert } from "chai";
+
+import { Connection } from "../../../src/connection/Connection";
+import {
+  closeTestingConnections,
+  createTestingConnections,
+  reloadTestingDatabases
+} from "../../utils/test-utils";
+import { Account } from "./entity/Account";
+import { AccountActivationToken } from "./entity/AccountActivationToken";
+
+describe("save child and parent entity", () => {
+  let connections: Connection[];
+  before(
+    async () =>
+      (connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+      }))
+  );
+  beforeEach(() => reloadTestingDatabases(connections));
+  after(() => closeTestingConnections(connections));
+
+  it("account property in accountActivationToken should not be null", () =>
+    Promise.all(
```

Comment:
please put return before Promise.all, tests are not finished because of this mistake